glean-core/src/metrics/event.rs

```diff
@@ -50,7 +50,7 @@ impl EventMetric {
     /// ## Arguments
     ///
     /// * `glean` - The Glean instance this metric belongs to.
-    /// * `timestamp` - A monotonically increasing timestamp, in nanoseconds.
+    /// * `timestamp` - A monotonically increasing timestamp, in milliseconds.
```

Comment:
There's also [another entry](https://github.com/mozilla/glean/blob/6880581aaf066fb7b2b55333e1c6def1a0e9669f/glean-core/src/event_database/mod.rs#L164) that needs to be fixed.